templates/rhpam73-prod-immutable-kieserver-amq.yaml

```diff
@@ -0,0 +1,1214 @@
+---
```

Comment:
For the master branch, this file name needs to have an rhpam74 prefix, not rhpam73.

Reply:
Also all "73 and 7.3" references in the file need to get updated as well.

Reply:
Old PR, changes made before the version update :)